src/Symfony/Component/JsonPath/Resources/bin/README.md

```diff
@@ -0,0 +1,15 @@
+# JsonPath Compliance Test Suite Fetcher
+
+This folder contains the tool to fetch the JSONPath compliance test suite.
+
+## Requirements
+
+* composer
+* make
+
+## Update the test suite
+
+To update to a newer version of the compliance test suite:
+
+1. Update the `reference` field in `composer.json` to the desired commit hash
```

Comment:
you need to update both the `reference` and the `version` field. Changing only the reference will break things for devs who already have a vendor folder locally, as running `composer update` will not install the new reference.
To be considered different, we need either to have a different `version`, or to have a version considered as a dev one and have a different `time`